ledger-service/http-json/src/main/scala/com/digitalasset/http/endpoints/CreateAndExercise.scala

```diff
@@ -47,16 +47,31 @@ private[http] final class CreateAndExercise(
         cmd <-
           decoder
             .decodeCreateCommand(reqBody, jwt, toLedgerId(jwtPayload.ledgerId))
-            .liftErr(InvalidUserInput): ET[domain.CreateCommand[ApiRecord, TemplateId.RequiredPkg]]
+            .liftErr(InvalidUserInput): ET[
```

Comment:
Why is the annotation needed

Reply:
I believe this one serves two purposes:

1. widen the error type so `flatMap` works
2. "annotate" the `cmd` binding, since [writing `cmd: typ <-` would be a filter, not an annotation](https://discuss.daml.com/t/pattern-always-adds-withfilter-scala-ritual/1463)